lib/resin.coffee

```diff
@@ -84,4 +84,60 @@ module.exports = getSdk = (opts = {}) ->
 		pine
 	}
 
-	return mapValues(sdkTemplate, (v) -> v(deps, opts))
+	sdk = mapValues(sdkTemplate, (v) -> v(deps, opts))
```

Comment:
BTW let's make this gem from me a little bit more readable by changing `v` to `moduleFactory` :)